src/schema/repair-request.ts

```diff
@@ -1,5 +1,29 @@
 import { z } from "zod";
 
+import { paginationSchema } from "@/lib/pagination";
+import prisma from "@/lib/prisma";
+
+const stringOrArray = z.union([
+  z.literal("").transform(() => undefined),
+  z.string().transform((s) => [s]),
+  z.array(z.string())
+]);
+
+export const getRepairRequestSchema = paginationSchema.extend({
+  id: z.string(),
+  sortKey: z
+    .string()
+    .refine((value) => value in prisma.repairRequest.fields, {
+      message: "Incorrect value for sortKey"
+    })
+    .optional(),
```

Comment:
Nice, but are we allowing sorting on all fields?

Reply:
The backend allows it, but the frontend only exposes what is logical. I can change this if needed.

Reply:
Ideally, the backend should be the one to make the decision on what is “logical”, not the frontend.